### DISCUSSION TITLE
User Permissions

### DISCUSSION THREAD

**Discussion by brucebailey** [MasaCMS/MasaCMS#85] [Q&A]
My over arching goal is to put an approval chain in place, but I can't even seem to get a new user added in general. 

Add a standard system user and system group of admin. Not super admin, but active.

User can login and view content, change content, but not publish or save content in any way.  I have to missing something tiny.

<img width="597" alt="Screen Shot 2022-05-10 at 12 23 44 PM" src="https://user-images.githubusercontent.com/913250/167679572-37bc67fc-5d14-4825-b746-bf9f47becda5.png">
<img width="591" alt="Screen Shot 2022-05-10 at 12 23 34 PM" src="https://user-images.githubusercontent.com/913250/167679593-bad38edb-7cac-4880-8a5e-3f3224698d27.png">

**Comment by brimard** [comment]
A couple of other places to check:

- Under site settings-->permissions, make sure group has content manager access (for tree/admin view)
- on content itself, under actions-->permissions via full edit or under the three dot nav in tree view-->permissions, make sure the group has 'editor' permissions. 

Sounds like they may be set 'author' at the content level, which does let you save a 'draft' I believe, but wouldn't let you publish or save the content.

Not sure if this will fix your issues, but a couple of basic things to check.

**Comment by brucebailey** [comment]
Thank you @brimard 

I checked everything over and it seems like I had everything in place and it still would not let the user save changes.  Then I, by chance, tried editing on the front-end and it worked like a champ.  So the settings I have now just won't let the user edit on the back-end.

To tie into that, this seems to cause issues with approval chains b/c you can't see drafts or history on the front end.
